platform/darwin/src/MGLStyle.h

```diff
@@ -270,66 +270,20 @@ MGL_EXPORT
 + (NSURL *)satelliteStreetsStyleURLWithVersion:(NSInteger)version;
 
 /**
- Returns the URL to the current version of the
+ Returns the URL to version 2 of the
  <a href="https://www.mapbox.com/blog/live-traffic-maps/">Mapbox Traffic Day</a>
  style.
 
- Traffic Day color-codes roads based on live traffic congestion data. Traffic
- data is currently available in
- <a href="https://www.mapbox.com/api-documentation/pages/traffic-countries.html">these select countries</a>.
-
- @warning The return value may change in a future release of the SDK. If you use
-    any feature that depends on a specific aspect of a default style – for
-    instance, the minimum zoom level that includes roads – use the
-    `-trafficDayStyleURLWithVersion:` method instead. Such details may change
-    significantly from version to version.
- */
-+ (NSURL *)trafficDayStyleURL;
-
-/**
- Returns the URL to the given version of the
- <a href="https://www.mapbox.com/blog/live-traffic-maps/">Mapbox Traffic Day</a>
- style as of publication.
-
- Traffic Day color-codes roads based on live traffic congestion data. Traffic
- data is currently available in
- <a href="https://www.mapbox.com/api-documentation/pages/traffic-countries.html">these select countries</a>.
-
- @param version A specific version of the style.
  */
-+ (NSURL *)trafficDayStyleURLWithVersion:(NSInteger)version;
++ (NSURL *)trafficDayStyleURL __attribute__((deprecated("Create an NSURL object with the string “mapbox://styles/mapbox/traffic-day-v2”.")));
```

Comment:
This change renames the selector from `-trafficDayStyleURLWithVersion:` to `-trafficDayStyleURL`. This is a backwards-incompatible change. Consider retaining the original method name but deprecating it nonetheless.

Reply:
@1ec5 I'm deprecating `+ (NSURL *)trafficDayStyleURL;` and deleting `trafficDayStyleURLWithVersion:(NSInteger)version` is that ok or should I keep the later?

Reply:
We shouldn’t delete anything without a major version bump. So we’ll need to keep both symbols and mark them both as deprecated.